lib/src/builder/getAbiDeclaration.ts

```diff
@@ -0,0 +1,21 @@
+import { functionDefinition } from '../Types/AbiTypes';
+import { getPath, getName, getDetails } from './utils'
+import { Output } from '../io';
+
+export const getAbiDeclaration = (abi: any, interfaceName: string, outputFile: string): string => {
+ let abiTypings = '';
+ let connectedAbiTypings = '';
+ let parameterCount = 0;
+ abi.forEach((functionDefinition: functionDefinition) => {
+	if(functionDefinition.type !== 'event' && functionDefinition.type !== 'fallback'){
```

Comment:
consider destructuring `functionDefinition` before the `if` statement to make it more legible